contrib/firmware_packager/simple_client.py

```diff
@@ -62,13 +62,23 @@ def parse_args():
         help="Reinstall payloads(default False)",
     )
     parser.add_argument(
-        "command", choices=["get-devices", "get-details", "install"], help="What to do"
+        "command", choices=["get-devices", "get-details", "install", "refresh"], help="What to do"
     )
     parser.add_argument('cab', nargs='?', help='CAB file')
     parser.add_argument('deviceid', nargs='?', help='DeviceID to operate on(optional)')
     args = parser.parse_args()
     return args
 
+def refresh(client):
+    """Uses fwupd client to refresh metadata"""
+    remotes = client.get_remotes()
+    client.set_user_agent_for_package("simple_client", client.get_daemon_version())
```

Comment:
This is meant to be the front-end version, not the Daemon version.

Reply:
We don't exactly have a version associated with `simple_client.py`.  Thoughts what to put there?

Reply:
It needs to be the version from the top-level meson.build, I guess we just add a VERSION to the unbuilt file.